src/Assertion/AssertionPluginManager.php

```diff
@@ -31,35 +33,5 @@
  */
 class AssertionPluginManager extends AbstractPluginManager
```

Comment:
final?

Reply:
can't mock, I've tried to use AbstractPluginManager, but didn't work either

Reply:
why didn't it worked? Also, I think ContaienrInterface would be good enough no? not sure where in tests it is problem. Can you point where it is problem maybe we can think something?

Reply:
because of: TypeError: Argument 3 passed to ZfcRbac\Service\AuthorizationService::__construct() must be an instance of ZfcRbac\Assertion\AssertionPluginManager, instance of Mock_ContainerInterface_660c8ef8 given, called in .../test/Service/AuthorizationServiceTest.php on line 284

Reply:
cant we then just use 
```php
$authorizationService = new AuthorizationService($rbac, $roleService, new AssertionPluginManager);
```
sorry just guess :D